themes/_default.yaml

```diff
@@ -1,4 +1,6 @@
 generic:
+  breakpoints:
+    small-max: '768px'
```

Comment:
Breakpoints should not be part of the ui kit. Can you pls provide your solution without using it? :)